Hardware/12-Wifi_Programming_Tutorial.md

```diff
@@ -100,36 +116,59 @@ Hook up the FTDI friend to the Wifi Shield, don't power the Wifi shield through
 
 Now upload using either the Arduino IDE or a custom make solution.
 
-## Sending Data to Wifi Shield
+## Connecting to the Wifi Shield
```

Comment:
I think this part should go on the getting started guide.

Reply:
I put it in the OpenBCI_Wifi_Server.md